### ISSUE TITLE
Big thanks to everyone for your contributions! We're giving 25 GB Neos storage boost to everyone*

### ISSUE THREAD

**Issue by Frooxius** [Frooxius/NeosLocale#50]
First I wanted to say, thanks so much everyone for helping translate Neos into so many different languages! I didn't expect to see this many so soon, but it's really cool to see Neos in different locales (even if I can't understand them) :D 

As small show of thanks we're giving permanent 25 GB storage boost to all contributors. If you have contributed, I recommend to put your Neos username into the "Authors" list in the translation file.

I've already awarded all the existing ones, you should receive a thanks message in Neos and see your storage go up by 25 GB. If you were missed, please let me know here!

**Comment by FreakyWaves** [comment]
Can I have it ? I made changes to around 30 lines, is it enough ?

**Comment by Frooxius** [comment]
@FreakyWaves Yes that's sufficient! Did you add yourself to the Authors list on the locale files? What is your Neos username?

**Comment by AdmiralSoap** [comment]
Such a wonderful idea as a little extra incentive for translators!
At first I was super worried about if people might abuse it, but glad it's being monitored.

Really enjoying continuing to translate Neos to Finnish!

**Comment by FreakyWaves** [comment]
Well I am currently translating more entries to french, but my username is the same

**Comment by Avunia** [comment]
Heya!
Seems I have been missed (or it was sent to my old account that I cannot access for some reason)
I was helping with the German translation (`de.json`) and got added as a contributor there (thanks btw 😃)
Username in Neos is `Avunia Takiya`

**Comment by foxy-1423** [comment]
hi, I created the Norwegian translation, well I'm only halfway done, but... is that enought?

**Comment by Frooxius** [comment]
@Avunia Sorry for missing you, should be granted now!
@foxy-1423 Yes, it's sufficient, thank you!

**Comment by alexderpyfox** [comment]
i added translations to nl locale was forked from jena i added the tutorial translation  and inspector translation 
and a few missing lines in between to the nl locale

**Comment by alexderpyfox** [comment]
Does that still count for 25gb even tho I forked if from jena  since that was at the time the latest since it wasn't merged yet but I did about 200 lines 😯 jena can confirm

**Comment by Frooxius** [comment]
Yes that counts! It should be granted now. Sorry for the delay!

**Comment by shadowpanther** [comment]
I started to add new strings to the Russian locale and tried my best at fixing the overall quality and consistency. Added myself to the Authors. Username in the game is "Shadow Panther".

**Comment by kazu0617** [comment]
Will I be entitled to receive Boost?
I've sent out several PRs for translation and conflict resolution. It's not a lot, considering the number of lines...

私はBoostを受け取る権利はもらえるのでしょうか？
何回か翻訳のPRをだしたり、コンフリクトの解消を行ったりしています。行数で考えると決して多いとは言えませんが…

**Comment by MeronGit** [comment]
Hello, I am also a new contributor and helped writing tutorial to Estonian (and some more lines) :)

**Comment by 3x1t-5tyl3** [comment]
Hello~
Did changes to 152 lines recently and am working on more for the german translation which aren't proper and grammatical fixes (should be by tuesday).
Is that sufficient or is more required?

**Comment by Frooxius** [comment]
Yes that's definitely sufficient thank you! I've granted the space, if I missed anyone, please let me know!

**Comment by Inferus25** [comment]
Hello! 
I added 63 new translations and fixed many other lines to the eo.json. Is that enough to get the boost?
I will continue helping maintain that locale, clearly :)

**Comment by Ruzert** [comment]
Hello! I added a bunch of tweaks and fixes to the Spanish translation when there were about 700 lines, and have been maintaining it ever since. Could I get the boost?

**Comment by marsmaantje** [comment]
HI,
Having translated all the new strings for the streaming room to dutch and fixed up the file in general, would this be enough? Or do I need to do more. (will probably continue contributing regardless)

**Comment by Frooxius** [comment]
Hello! Yes, the spaces should be all granted now, please let me know if I missed anyone!

Also if possible, please include your Neos username if it's different from here, sometimes I'm not sure if I got the right one. 

@Ruz-eh Is your Neos username "Ruz"?

**Comment by Ruz-eh** [comment]
@Frooxius Yes it is! Thanks!

**Comment by Ducky07** [comment]
I would like to request storage for Toni Kat (@AirRide91), since he has been helping out a lot with the Finnish translations.

**Comment by Frooxius** [comment]
@Ducky07 Of course! Is it "Toni Kat" on Neos too or just a name coincidence? Couldn't find AirRide91 on Neos.

**Comment by shiftyscales** [comment]
> 
> 
> @Ducky07 Of course! Is it "Toni Kat" on Neos too or just a name coincidence? Couldn't find AirRide91 on Neos.

Yes, `AirRide#6519` on Discord is `Toni Kat` (U-Toni-Kat) on Neos.

**Comment by Frooxius** [comment]
Thanks, just making sure! Ok should be granted now!

**Comment by Titancube** [comment]
Hi! May I request storage for myself and other contributors in Korean translations?

- Titancube (U-Titancube)
- R3C0D3r (U-R3C0D3r)
- LUA (U-LUA)
- Guide (U-Guide)
- Holy_Water (U-Holy-Water)

MirPASEC told me he got his storage long ago so I excluded him from the list.

**Comment by InnocentThief** [comment]
Hello. can you add the space for me too. Added all the new tutorial translations and made a lot of grammar/typo corrections

**Comment by Frooxius** [comment]
Should be all granted! Thank you for your contributions!

**Comment by yellowcrash10** [comment]
I would like to request storage for myself (U-yellowcrash10) for my contributions to the Esperanto locale (386 changed/added lines if I am reading GitHub's stats correctly).

**Comment by Frooxius** [comment]
Granted! Sorry for taking so long, I missed the new posts on this issue. Thank you for your contributions!

**Comment by aragubas** [comment]
I just made a Brazilian Portuguese locale for Neos, it took me 5 hours of editing the JSON in VSCode lol, is that still valid?
